seahorn/jobs/hash_table_create/aws_hash_table_create_harness.c

```diff
@@ -0,0 +1,55 @@
+#include <aws/common/hash_table.h>
+#include <aws/common/private/hash_table_impl.h>
+
+#include <seahorn/seahorn.h>
+
+#include <config.h>
+#include <hash_table_helper.h>
+#include <utils.h>
+
+int main(void) {
+  /* data structure */
+  struct aws_hash_table map;
+  initialize_bounded_aws_hash_table(&map, MAX_TABLE_SIZE);
+  assume(aws_hash_table_is_valid(&map));
+  sea_printf("entry_count init:", map.p_impl->entry_count);
+  // use nondeterminism
+  // map.p_impl->equals_fn = nondet_equals;
```

Comment:
these fields should be initialized as in commented lines. Even if they are later not used due to override

Reply:
they are initialized in helper function `initialize_bounded_aws_hash_table`

Reply:
then remove the commented lines